roles/nova-common/templates/etc/nova/policy.json

```diff
@@ -289,7 +289,7 @@
     "os_compute_api:os-admin-password": "rule:admin_or_owner",
     "os_compute_api:os-admin-password:discoverable": "@",
     "os_compute_api:os-aggregates:discoverable": "@",
-    "os_compute_api:os-aggregates:index": "rule:admin_api",
```

Comment:
add this is for command: "openstack hypervisor show compute-0".
Since for data migration, cloud_admin may want to view the hypervisor status to determine the target hypervisor.
https://docs.openstack.org/developer/nova/sample_policy.html said this policy is related to " List all aggregates". So is it ok to let cloud_admin have this privilege?